docs/additional-configuration.adoc

```diff
@@ -351,3 +351,25 @@ spec:
 ----
 
 For documentation on Runtime Classes, see https://kubernetes.io/docs/concepts/containers/runtime-class/
+
+## Configuring the Webhook deployment
+The `devworkspace-webhook-server` deployment can be configured in the global DevWorkspaceOperatorConfig (DWOC). The configuration options include: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas[replicas], https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/[pod tolerations] and https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector[nodeSelector]. 
+
+These configuration options exist in the **global** DWOC's `config.webhook`  field:
+
+```YAML
+apiVersion: controller.devfile.io/v1alpha1
+kind: DevWorkspaceOperatorConfig
+metadata:
+  name: devworkspace-operator-config
+  namespace: $OPERATOR_INSTALL_NAMESPACE
+config:
+ webhook:
+   nodeSelector:  <string, string>
+   tolerations: <[]tolerations>
+   replicas: <int32>
+```
+**Note:** In order for the `devworkspace-webhook-server` configuration options to take effect:
+
+- You must place them in the https://github.com/devfile/devworkspace-operator?tab=readme-ov-file#global-configuration-for-the-devworkspace-operator[global DWOC], which has the name `devworkspace-operator-config` and exists in the namespace where the DevWorkspaceOperator is installed. If it does not already exist on the cluster, you must create it.
+- You'll need to terminate the `devworkspace-controller-manager` pod and restart it so that the `devworkspace-webhook-server` deployment can be adjusted accordingly.
```

Comment:
Can we update to:
```suggestion
- You'll need to terminate the `devworkspace-controller-manager` pod so that the replicaset can recreate it. The new pod will update the `devworkspace-webhook-server` deployment.
```

it's to make it clear that the user doesn't have to restart anything, all the user has to do is terminate the pod and wait until it's recreated

Reply:
Good call, done :)